### DISCUSSION TITLE
ezdxf.path.rect SyntaxError

### DISCUSSION THREAD

**Discussion by nhuffer** [mozman/ezdxf#1026] [Issues]
Why do I get this?

 ezdxf.path.rect(width: float = 1, height: float = 1, transform: Matrix44 = None)
                         ^
SyntaxError: invalid syntax

**Comment by ghost** [comment]
it might be related to the equal signs. sometimes in python it gives you a syntax error if you use "=" instead of "==" in your code. try using the double equals signs instead of the single one. i see that this post is a bit more than a month old, so idk if you haven't debugged it yet.

hope this helps :)

**Comment by mozman** [comment]
A method/function call cannot contain  type annotations. Type annotations are only allowed in method/function declarations.
